tests/closures-as-members-repeated-reallocation.phpt

```diff
@@ -0,0 +1,31 @@
+--TEST--
+Testing closure members reallocation
+--DESCRIPTION--
+This test verifies that closures can be set as members repeatedly
+--FILE--
+<?php
+class TestClosure extends \Threaded implements \Collectable {
+    protected $closure;
+
+    function __construct( $closure) {
+        $this->closure = $closure;
+    }
+
+    public function run() {
+        ($this->closure)();
+    }
+}
+
+$count = 0;
+$worker = new Worker();
+$worker->start();
+
+while ($count++ < 5) {
```

Comment:
I just ran this test case on an arm64 binary (without this bug fix) and it doesn't reproduce with a low number due to ???. I suggest increasing this to something like 20 (which does reproduce the crash).